testgrid/tgrun/pkg/scheduler/types/types.go

```diff
@@ -35,8 +35,8 @@ type OperatingSystemImage struct {
 }
 
 type Instance struct {
-	InstallerSpec    InstallerSpec
-	UpgradeSpec      *InstallerSpec
+	InstallerSpec    InstallerSpec  `json:"installerSpec" yaml:"installerSpec"`
+	UpgradeSpec      *InstallerSpec `json:"upgradeSpec,omitempty" yaml:"upgradeSpec,omitempty"`
 	UnsupportedOSIDs []string
```

Comment:
I think it would be nice to have support for this too, then we could add some tests for RKE, but not essential in this PR.

Reply:
what do you mean?

Reply:
`UnsupportedOSIDs`

Reply:
ohhhh
yeah, I can add that